cli/connect/binary.go

```diff
@@ -0,0 +1,92 @@
+package connect
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/apex/log"
+	"github.com/c-bata/go-prompt"
+	"github.com/tarantool/go-tarantool"
+)
+
+func binaryConnect(console *Console) error {
+	var err error
+
+	console.binaryConn, err = tarantool.Connect(console.connOpts.Address, tarantool.Opts{
+		User: console.connOpts.Username,
+		Pass: console.connOpts.Password,
```

Comment:
AFAIR you use timeouts for unix sockets. I think you should use it for tcp sockets as well

Reply:
Added `common.EvalTarantoolConnTimeout`.